### ISSUE TITLE
BINANCE - SMART SWITCH?

### ISSUE THREAD

**Issue by iceryan354** [whittlem/pycryptobot#25]
great work and communication, thanks for that.
 forgot to ask - is binance supporting a smart switching?

doesnt seem to be indicated in the documentation whether yes or no, just absent from config...

thanks

**Comment by rajkumarboppana** [comment]
smart switching is working from 1h to 15m in Binance also

**Comment by whittlem** [comment]
Yes, all the features for Coinbase Pro are replicated for Binance.
